packages/types/src/event.ts

```diff
@@ -41,6 +41,7 @@ export interface Event {
   type?: EventType;
   spans?: Span[];
   measurements?: Measurements;
+  tracestate?: string;
```

Comment:
not cool, it makes it really confusing for somebody reading the code. 
How would one know that this is just some temp thing.
I feel even stronger now that we should hide all temp things before one clearly temp object.

Reply:
See https://github.com/getsentry/sentry-javascript/pull/3092#discussion_r543593743.